### PR TITLE
Optionally prevent virtual levels from effecting total level

### LINKED ISSUE #5878: Toggle option for total level affected by virtual level
Currently the virtual level plugin also adds them to your total level, is there any way to add a toggle option so it does not change that? 
virtual on
virtual on1

### PR DESCRIPTION
**Pull request by beaumitch**
Resolves #5878

virtuallevels1

(Used a mock total level above to show virtual total, I don't have any skills past 99)

virtuallevels2

### CODE REVIEW

File: runelite-client/src/main/java/net/runelite/client/plugins/virtuallevels/VirtualLevelsPlugin.java [hunk 1]
```diff
@@ -110,6 +131,12 @@ public void onScriptCallbackEvent(ScriptCallbackEvent e)
 		}
 	}
 
+	@Provides
```
Comment (Nightfirecat) [inline runelite-client/src/main/java/net/runelite/client/plugins/virtuallevels/VirtualLevelsPlugin.java]: This should be placed next to the `@Override` method(s) in the file.

File: runelite-client/src/main/java/net/runelite/client/plugins/virtuallevels/VirtualLevelsConfig.java [hunk 2]
```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (c) 2018, Joshua Filby <joshua@filby.me>
```
Comment (Nightfirecat) [inline runelite-client/src/main/java/net/runelite/client/plugins/virtuallevels/VirtualLevelsConfig.java]: Neither of us wrote this file, you should use your own copyright attribution for this.

**Comment by beaumitch** [comment]
> 
> 
> LGTM otherwise

Updated, thanks for review.

File: runelite-client/src/main/java/net/runelite/client/plugins/virtuallevels/VirtualLevelsConfig.java [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+import net.runelite.client.config.Config;
+import net.runelite.client.config.ConfigGroup;
+import net.runelite.client.config.ConfigItem;
+
+@ConfigGroup("virtuallevels")
+public interface VirtualLevelsConfig extends Config
+{
+	@ConfigItem(
+			keyName = "virtualTotalLevel",
```
Comment (Nightfirecat) [inline runelite-client/src/main/java/net/runelite/client/plugins/virtuallevels/VirtualLevelsConfig.java]: Reduce the indent by one tab for each of these lines.

File: runelite-client/src/main/java/net/runelite/client/plugins/virtuallevels/VirtualLevelsPlugin.java [hunk 4]
```diff
@@ -25,33 +25,45 @@
...
+	description = "Shows virtual levels (beyond 99) and virtual skill total on the skills tab.",
 	tags = {"skill", "total", "max"},
 	enabledByDefault = false
 )
 public class VirtualLevelsPlugin extends Plugin
 {
 	private static final String TOTAL_LEVEL_TEXT_PREFIX = "Total level:<br>";
 
+	@Inject
+	private VirtualLevelsConfig virtualLevelsConfig;
```
Comment (Nightfirecat) [inline runelite-client/src/main/java/net/runelite/client/plugins/virtuallevels/VirtualLevelsPlugin.java]: For brevity

```suggestion
	private VirtualLevelsConfig config;
```

**Pull request merged**